source/javascripts/components/unified-editor/WorkflowCard/components/ChainedWorkflowList.tsx

```diff
@@ -153,43 +153,34 @@ const ChainedWorkflowList = ({ placement, containerProps, parentWorkflowId, ...a
     },
   });
 
-  useLayoutEffect(() => {
+  useEffect(() => {
     setSortableItems(initialSortableItems);
   }, [initialSortableItems]);
 
-  if (sortableItems.length === 0) {
-    return (
-      <Box
-        gap="8"
-        display="flex"
-        overflow="clip"
-        flexDir="column"
-        transition="all 250ms"
-        height={active ? 74 : 0}
-        mt={!active && isAfterRun ? -8 : 0}
-        mb={!active && isBeforeRun ? -8 : 0}
-        onTransitionEnd={() => measureDroppableContainers(droppableContainers.toArray().map((d) => d.id))}
-        {...containerProps}
-      >
-        {isAfterRun && <Icon name="ArrowDown" size="16" color="icon/tertiary" alignSelf="center" />}
-        <Droppable placement={placement} parentWorkflowId={parentWorkflowId} />
-        {isBeforeRun && <Icon name="ArrowDown" size="16" color="icon/tertiary" alignSelf="center" />}
-      </Box>
-    );
-  }
+  const isEmpty = sortableItems.length === 0;
 
   return (
     <SortableContext
       disabled={!isSortable}
       strategy={verticalListSortingStrategy}
       items={getSortableItemUniqueIds(sortableItems)}
     >
-      <Box display="flex" flexDir="column" gap="8" {...containerProps}>
-        {isAfterRun && <Icon name="ArrowDown" size="16" color="icon/tertiary" alignSelf="center" />}
+      <Box
+        gap={8}
+        display="flex"
+        flexDir="column"
+        mt={isEmpty && isAfterRun ? -8 : 0}
+        mb={isEmpty && isBeforeRun ? -8 : 0}
+      >
+        {isAfterRun && !isEmpty && <Icon name="ArrowDown" size="16" color="icon/tertiary" alignSelf="center" />}
+        {isAfterRun && isEmpty && <Droppable placement={placement} parentWorkflowId={parentWorkflowId} />}
```

Comment:
Why is the `isAfterRun` before the `ChainWorkflowCard` and the `isBeforeRun` after it?